web/fido_app/models.py

```diff
@@ -132,6 +132,7 @@ class Interaction(db.Model):
     session_token = db.Column(db.String(40), db.ForeignKey('session.token'))
     element = db.Column(db.String(32), nullable=False)
     event = db.Column(db.Enum('focus', 'click', 'submit', 'load', validate_strings=True), nullable=False)
+    login_method = db.Column(db.Integer, default=0) # 0 is not logging in, 1 is webauthn, and 2 is password
```

Comment:
I know that it would be functionally the same, but is there any reason that a field like this isn't an Enum? It would be nice for interpretability.